src/ngf/main.c

```diff
@@ -179,7 +179,7 @@ main (int argc, char *argv[])
     AppData app;
 
     memset (&app, 0, sizeof (app));
-    app.default_loglevel = N_LOG_LEVEL_ERROR;
+    app.default_loglevel = N_LOG_LEVEL_DEBUG;
```

Comment:
You should not change the default log level.

Reply:
Thanks, changed. Let me know if you have suggestions for improvement for the other changes too. For example, flag in the config file for caching the effects, or detection of the rumble effect maybe should be fixed in my kernel etc.